src/main/java/uk/gov/hmcts/reform/adoption/document/content/DocmosisTemplateProvider.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.adoption.document.content;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.adoption.common.config.DocmosisTemplatesConfig;
+import uk.gov.hmcts.reform.adoption.document.model.LanguagePreference;
+
```

Comment:
Add JUnit for this class. I can see you already added Junit for most of the classes, please make sure coverage should be 90% at least 
@rajatkgupta29 @abidfs @dilipsamrahmcts @gauravtomar-gh